pkg/api/graph/graph.go

```diff
@@ -353,10 +353,34 @@ func (g Graph) addEdges(edges []graph.Edge, fn EdgeFunc) {
 // node should be included.
 type NodeFunc func(g Interface, n graph.Node) bool
 
+// NewNodeFunc returns a new NodeFunc accepting the provided kinds of nodes
+func NewNodeFunc(kinds ...string) NodeFunc {
```

Comment:
`NodesOfKind`?  That makes the subgraph command read: `g.Subgraph(NodesOfKind(kind1, kind2), EdgesOfKind(kindA, kindB))`